### PR TITLE
fix(query): fix query descriptionText for s3 logging disabled kms rot…

### PR DESCRIPTION
**Pull request by tomk-orca**
…ation and iam policies

Closes #

**Proposed Changes**
-
-
-

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "queryName": "High KMS Rotation Period",
   "severity": "MEDIUM",
   "category": "Secret Management",
-  "descriptionText": "KMS rotation period should not surpass 365 days",
+  "descriptionText": "KMS Rotation Period should be greater than 365 days.",
```
Comment (rafaela-soares) [inline assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json:6]: Can you update also in `assets\queries\terraform\gcp\high_kms_rotation_period\metadata.json`, please?

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello, @tomk-orca!

Thank you so much for your contribution! 🚀 
We are very glad to see you here for the first time. Hope to see you more times! 😊 

I only have one request:

File: assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json (hunk 1 above)
Comment (tomk-orca) [reply assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json:6]: done

Comment (rafaela-soares) [reply assets/queries/ansible/gcp/high_kms_rotation_period/metadata.json:6]: Thank you so much!

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**